lib/chef/resource/lwrp_base.rb

```diff
@@ -74,13 +75,9 @@ def build_from_file(cookbook_name, filename, run_context)
 
         # Adds +action_names+ to the list of valid actions for this resource.
         # Does not include superclass's action list when appending.
+        # # TODO - do we want to support
```

Comment:
That comment is also wrong, witht this change it _does_ include superclass's actions because that's the default behavior of `allowed_actions`.